src/__tests__/ca/verify/index.test.ts

```diff
@@ -42,7 +42,21 @@ describe('verifySigningCertificate', () => {
   const signers: sigstore.CAArtifactVerificationOptions['signers'] = {
     $case: 'certificateIdentities',
     certificateIdentities: {
-      identities: [],
+      identities: [
+        {
+          issuer: 'https://github.com/login/oauth',
```

Comment:
This is nice! For registry, will it be possible to check issuer and oiods without checking for a particular `san`? (to validate part of the provenance contents against the cert)

Reply:
Ah I see this is required now, we can probably figure out what it should be during publish.